runtime/src/lib.rs

```diff
@@ -858,7 +858,9 @@ parameter_types! {
     pub const SubtensorInitialPruningScore : u16 = u16::MAX;
     pub const SubtensorInitialBondsMovingAverage: u64 = 900_000;
     pub const SubtensorInitialDefaultTake: u16 = 11_796; // 18% honest number.
-    pub const SubtensorInitialMinTake: u16 = 5_898; // 9%
+    pub const SubtensorInitialMinDelegateTake: u16 = 5_898; // 9%
+    pub const SubtensorInitialDefaultChildKeyTake: u16 = 11_796; // 18% honest number.
```

Comment:
I may remember this wrong from a voice conversation, but didn't we decide to make the default to be zero so that we don't deincentivise parent key delegation?

Reply:
done